api-server/src/main/resources/swagger.json

```diff
@@ -234,6 +234,48 @@
                     }
                 }
             },
+            "patch": {
+                "tags": [
+                    "enmasse_v1beta1",
+                    "addressspaces"
+                ],
+                "description": "patches (RFC6902) the specified AddressSpace",
+                "operationId": "patchesEnmasseV1alpha1NamespacedAddressSpace",
```

Comment:
Just a note here that it should probably be be V1beta1 in the name, but I understand this is an issue with the existing operationIds as well.

Reply:
@lulf  I'll fix all of these in one go, under a separate issue.